README.md

```diff
@@ -217,9 +190,9 @@ the project and its risk profile.
 Facilitator: Justin Cappos ([@JustinCappos](https://github.com/JustinCappos)),
 New York University
 
-Facilitator: Matthew Giassa ([@Iaxes](https://github.com/iaxes)), Cisco
+Facilitator: Andres Vega ([@anvega](https://github.com/anvega)), ControlPlane
 
-Co-chair representative: @lumjjb
+Co-chair representative: @sublimino
```

Comment:
I'm not sure it's the biggest problem, but in some ways it would be cleaner to have a co-chair who the facilitators didn't share an affiliation with.